libs/partners/mongodb-atlas/README.md

```diff
@@ -0,0 +1 @@
+# langchain-mongodb-atlas
```

Comment:
Should at least have Installation and Usage sections here.

Reply:
supplied!